docs/components/api/api-page-items.js

```diff
@@ -5,40 +5,57 @@ import ApiItemContents from './item-contents';
 import SectionWrapper from './section-wrapper';
 import Github from './github';
 import Feedback from '../feedback';
+import { toHtml } from '../../util/formatters';
 
 const apiFilterItems = require('../../util/api-filter-items.js');
 
 export default class ApiPageItems extends React.Component {
-    render() {
-        const pageDocSource = apiFilterItems(this.props.pageTitle);
-        const children = pageDocSource[0].members.static;
+    pageDocSource = apiFilterItems(this.props.pageTitle);
+    children = this.pageDocSource[0].members.static;
+    renderDescription = () => {
+        const description = this.pageDocSource[0].description || false;
+        if (description)
+            return <React.Fragment>{toHtml(description)}</React.Fragment>;
```

Comment:
I think you could remove the React.Fragment wrapper and return:

```suggestion
            return toHtml(description);
```